subworkflows/local/qc_bam.nf

```diff
@@ -59,6 +60,9 @@ workflow QC_BAM {
         SENTIEON_WGSMETRICS ( ch_bam_bai, ch_genome_fasta, ch_genome_fai, ch_intervals_wgs.map{ interval -> [[:], interval]} )
         SENTIEON_WGSMETRICS_Y ( ch_bam_bai, ch_genome_fasta, ch_genome_fai, ch_intervals_y.map{ interval -> [[:], interval]} )
 
+        // Check sex
+        NGSBITS_SAMPLEGENDER(ch_bam_bai, ch_genome_fasta, ch_genome_fai, "xy")
```

Comment:
We could consider making the method a parameter, ie `"xy"` becomes something like "ngsbits_samplegender_method" (or something that rolls of the tongue more easily), with the default set to `xy`.

Reply:
Done